File: VentasVB/cuCrudCargos.vb
Public Class cuCrudCargos
    Dim con As Conexion = New Conexion()
    Private Sub cuCrudCargos_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        dgvDatos.DataSource = con.datosCargos()
    End Sub

    Private Sub btnCrear_Click(sender As Object, e As EventArgs) Handles btnCrear.Click
        con.crearCargo(tbDescripcion.Text, Convert.ToInt32(nudSueldo.Value), idUsuario)
        dgvDatos.DataSource = con.datosCargos()
    End Sub

    Private Sub btnActualizar_Click(sender As Object, e As EventArgs) Handles btnActualizar.Click
        con.actualizarCargo(Convert.ToInt32(dgvDatos.SelectedRows(0).Cells(0).Value), tbDescripcion.Text, Convert.ToInt32(nudSueldo.Value), idUsuario)
        dgvDatos.DataSource = con.datosCargos()
    End Sub

    Private Sub btnEliminar_Click(sender As Object, e As EventArgs) Handles btnEliminar.Click
        con.eliminarTabla("Cargo", Convert.ToInt32(dgvDatos.SelectedRows(0).Cells(0).Value), idUsuario)
        dgvDatos.DataSource = con.datosCargos()
    End Sub

    Private Sub dgvDatos_CellClick(sender As Object, e As DataGridViewCellEventArgs) Handles dgvDatos.CellClick
        tbDescripcion.Text = dgvDatos.SelectedRows(0).Cells(1).Value.ToString()
        nudSueldo.Value = Convert.ToInt32(dgvDatos.SelectedRows(0).Cells(2).Value)
    End Sub
End Class

File: Ventas/cuCrudCargos.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Ventas
{
    public partial class cuCrudCargos : UserControl
    {
        Conexion con = new Conexion();
        public cuCrudCargos()
        {
            InitializeComponent();
            dgvDatos.DataSource = con.datosCargos();
        }

        private void btnCrear_Click(object sender, EventArgs e)
        {
            con.crearCargo(tbDescripcion.Text,Convert.ToInt32(nudSueldo.Value), global.id);
            dgvDatos.DataSource = con.datosCargos();
        }

        private void btnActualizar_Click(object sender, EventArgs e)
        {
            con.actualizarCargo(Convert.ToInt32(dgvDatos.SelectedRows[0].Cells[0].Value), tbDescripcion.Text, Convert.ToInt32(nudSueldo.Value), global.id);
            dgvDatos.DataSource = con.datosCargos();
        }

        private void btnEliminar_Click(object sender, EventArgs e)
        {
            con.eliminarTabla("Cargo",Convert.ToInt32(dgvDatos.SelectedRows[0].Cells[0].Value), global.id);
            dgvDatos.DataSource = con.datosCargos();
        }

        private void dgvDatos_CellClick(object sender, DataGridViewCellEventArgs e)
        {
            tbDescripcion.Text = dgvDatos.SelectedRows[0].Cells[1].Value.ToString();
            nudSueldo.Value = Convert.ToInt32(dgvDatos.SelectedRows[0].Cells[2].Value);
        }
    }
}
